### ISSUE TITLE
Error: missing severity rule

### ISSUE THREAD

**Issue by huy-nguyen** [palantir/tslint#2650]
### Bug Report

- __TSLint version__: 5.1.0
- __TypeScript version__: 2.3.1
- __Running TSLint via__: (pick one) CLI
`tslint --project tsconfig.json --config tslint.json --exclude '**/*.js'`

#### TypeScript code being linted

N/A

with `tslint.json` configuration:

```json
{
  "extends": ["tslint:latest", "tslint-react"],
  "rules": {
    "interface-over-type-literal": false,
    "quotemark": false,
    "ordered-imports": false,
    "object-literal-sort-keys": false,
    "arrow-parens": false,
    "one-variable-per-declaration": false,
    "only-arrow-functions": false,
    "semicolon": [true, "ignore-interfaces"],
    "no-console": [true, "log"],
    "member-ordering": false,
    "variable-name": [true, "ban-keywords", "allow-leading-underscore"],
    "member-access": false,
    "comment-format": false,
    "no-var-requires": false,

    "jsx-alignment": false,
    "jsx-curly-spacing": [true, "never"],
    "jsx-no-lambda": true,
    "jsx-no-multiline-js": true,
    "jsx-no-string-ref": true,
    "jsx-self-close": true
  }
}
```

#### Actual behavior

Should not complain about missing severity rule.

#### Expected behavior

```
node_modules/tslint/lib/runner.js:116
            throw error;
            ^

Error: Severity for rule 'no-empty-interface not found
    at Linter.lint (/Users/hnguyen1/Sites/new-intl-atlas/node_modules/tslint/lib/linter.js:109:23)
    at Runner.processFiles (/Users/hnguyen1/Sites/new-intl-atlas/node_modules/tslint/lib/runner.js:157:20)
    at Runner.run (/Users/hnguyen1/Sites/new-intl-atlas/node_modules/tslint/lib/runner.js:108:18)
    at Object.<anonymous> (/Users/hnguyen1/Sites/new-intl-atlas/node_modules/tslint/lib/tslint-cli.js:140:6)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
```

**Comment by javier-tarazaga** [comment]
Hi guys, 

I do have the same issue with the following rule: 

* **TSLint version:** 5.1.0
* **TypeScript version:** 2.3.1
* **Running TSLint via: (pick one)** CLI
tslint \"server/**/*.ts\
TypeScript code being linted

N/A

with tslint.json configuration:

```
{
  "extends": [
    "tslint:recommended",
    "tslint-eslint-rules"
  ],
  "rules": {
    // "no-unused-vars": true, Still deprecated but will change soon https://github.com/palantir/tslint/issues/1481
    "arrow-parens": [
      true,
      "ban-single-arg-parens"
    ],
    "quotemark": [
      true,
      "single",
      "jsx-double"
    ],
    "ter-indent": [
      true,
      2,
      {
        "FunctionExpression": {
          "parameters": 1,
          "body": 1
        }
      }
    ]
  }
}
```

Error: Severity for rule 'trailing-comma not found

**Comment by gargantuan** [comment]
I have the same problem with `Severity for rule 'object-literal-sort-keys not found`

**Comment by nchen63** [comment]
fixed in #2516

**Comment by gargantuan** [comment]
@nchen63 - I still have the problem in tslint@5.1.0. #2516 was merged 23 days ago, this issue was raised 5 days ago. It doesn't appear to be fixed, unless I'm missing something?

**Comment by huy-nguyen** [comment]
@gargantuan The commit that merges in #2516 (617e705330a4e64f84d8d2759f684437203b75a8) is [one commit behind](https://github.com/palantir/tslint/compare/d8db246...617e705330a4e64f84d8d2759f684437203b75a8) the v5.1 release (d8db246) so it just missed the last release. 

@nchen63 I think the community would really appreciate if you can make a patch release that includes this fix.

**Comment by adidahiya** [comment]
v5.2 is coming very soon: https://github.com/palantir/tslint/pull/2666
